notebooks/visualization-03-cartopy.md

```diff
@@ -408,6 +408,69 @@ sst.plot(ax=ax, transform=ccrs.PlateCarree(),
          vmin=2, vmax=30, cbar_kwargs={'shrink': 0.4})
 ```
 
+## Add OGC WMS and WMTS services to cartopy 
+
++++
+
+A lot of online map services are available, which are capable of providing background layers to a map similar as these are used in online applications. The [OGC WMS/WMTS](https://www.ogc.org/standards/wms) are important standards by which a lot of map layers are made available. 
+
+A (non-official, but very comprehensive) list of services is made available by Michel Stuyts, see https://wms.michelstuyts.be. It is always a matter of finding the correct URL and the `layer_name` for a given service. 
+
+As an example, consider the Orthophoto WMS (provided by the National Geographic Institute): 
+
+- WMS: https://wms.ngi.be/inspire/ortho/service
+- GetCapabilities: https://wms.ngi.be/inspire/ortho/service?REQUEST=GetCapabilities&SERVICE=WMS
```

Comment:
Clicking those links will open a file, that's expected? (they don't have a web client showing the result of a query?)

Reply:
Indeed, it is just as this service provides it, we'll have to deal with it as such. I would keep it here?

Reply:
Yep